### DISCUSSION TITLE
Rejection

### DISCUSSION THREAD

**Discussion by Vanshika110** [community/community#135345] [GitHub Education]
### Hi everyone,

I have corrected all points, firstly I have uploaded from my device the  my id card with date on it but it got rejected then again I uploaded my second I'd card it also got rejected .Even I tried to upload my result also but no luck
They said to direct click from camera but my camera is not turning back I don't know why, and I tried in other mobile phone also due to which I am unable to take photo on the spot, I am tired of submitting the form but it isn't accepting please tell me what to do..

**Comment by sachetsubedi** [comment]
You can take photo from your front camera using your laptop or phone ensuing the details on the cards are visible. You may also include yourself for better changes of acceptance. This ensures you are not using a fake id card made in Photoshop or something.

**Comment by davevad93** [comment]
Hi @Vanshika110 ,

Take a look at [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352), you may find useful information about your rejection.

The [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student) state the following about the required student documentation:

- ***Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.***

So if possible try to provide some of the above documentation in addition to your ID card.

Also make sure that your documentation shows a current date in either `DD/MM/YYYY` or `MM/DD/YYYY` format, the academic year alone may not be enough to prove your current enrollment.

**Comment by Vanshika110** [comment]
I have attached result one time also
